.github/workflows/ci_cd-untp-playground.yml

```diff
@@ -63,30 +67,31 @@ jobs:
       id-token: write
       contents: read
 
-    name: Deploy to Test
+    name: Deploy to Prod
     environment:
       name: production
       url: https://test.uncefact.org/untp-playground
 
-    env:
-      STACK_NAME: prod
-
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
 
-      # TODO: setup a role in AWS account
-      - name: Configure AWS Credentials
+      - name: aws_login_uncefact
         uses: aws-actions/configure-aws-credentials@v4
         with:
-          aws-access-key-id: ${{ secrets.PULUMI_AWS_SECRET_KEY_ID }}
-          aws-region: ap-southeast-2
-          aws-secret-access-key: ${{ secrets.PULUMI_AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: arn:aws:iam::651690125008:role/tests_untp
```

Comment:
I think, should the role be a variable?

Reply:
changed to a repo secret `AWS_ROLE_TO_ASSUME`